web/src/components/IdeConsole/IdeConsole.js

```diff
@@ -1,17 +1,35 @@
-import { useContext } from 'react'
+import { useContext, useEffect } from 'react'
 import { IdeContext } from 'src/components/IdeToolbarNew'
 
 const IdeConsole = () => {
   const { state } = useContext(IdeContext)
+  useEffect(() => {
+    const element = document.querySelector('.console-tile .mosaic-window-body')
+    if (element) {
+      element.scrollTop = element.scrollHeight - element.clientHeight
+    }
+  }, [state.consoleMessages])
```

Comment:
This effect scrolls the console to the bottom whenever there is a new message. Users can still scroll up to look at history, it just snaps back down next message. We might want to add a checkbox to turn off/on this functionality, but I think this is a good default and unless someone complains about it, might as well stay as is.

Reply:
Hey nice! So few lines of code. Yeah that should be configurable in future: that's one of those less-technical bits I can deal with for you coming up, so you don't have to waste time on it.